### ISSUE TITLE
On Updated ScanNet Results

### ISSUE THREAD

**Issue by kentang-mit** [DylanWusee/pointconv#22]
Dear authors,

I have noticed that you submitted a version to ScanNet benchmark recently, and the result (mIoU=66.6%) is significantly better than the CVPR'19 version (55.6%). Is it possible to describe what change you have made to the original CVPR'19 model to achieve such large performance boost? Thanks a lot for your response.

Best,
Haotian

**Comment by DylanWusee** [comment]
Hi, 

Thank you for your interest in our work.

There are several changes that I made to achieve better performance as bellow:

1. Train with more dense point clouds. Originally, we use N = 8192 points, now we use N <= 100000 points.

2. We remove Batch Normalization. We find that batch normalization can give more overfitting in our model.

3. we use a much deeper network structure. We add 3~5 residual blocks after each downsampling.

We will try to release the related code and the trained model in pytorch as soon as possible.
Please refer: https://github.com/DylanWusee/pointconv_pytorch

Thank you very much.
Wenxuan

**Comment by kentang-mit** [comment]
Hi Wenxuan,

Thanks a lot for your helpful response. The observation 2 seems to be very interesting~ I'm closing this issue because it has solved my problem.

Best,
Haotian

**Comment by linhaojia13** [comment]
HI, @DylanWusee , do you add RGB channels in the input in your updated model?
